package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/226114575-e59b0213-bb51-4d8e-a132-49b41d7a7ea1.png)

- [ ] Kindly replace this command shown in the image above :arrow_up: with `npm install` to make your readme file descriptive and useful :blush:

Reply:
Thanks.